feed/feed.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
```

Comment:
The file name should be `index.html` and not `feed.html`

Reply:
Fixed